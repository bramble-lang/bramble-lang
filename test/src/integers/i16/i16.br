fn my_main() -> i64 {
    let x:i16 := 4i16;
    let y:i16 := test();
    let z:i16 := test2(6000i16);

    project::std::io::writei16ln(x);
    project::std::io::writei16ln(y);
    project::std::io::writei16ln(z);
    
    return 0;
}

fn test() -> i16 {
    return 5i16;
}

fn test2(i: i16) -> i16 {
    return i;
}