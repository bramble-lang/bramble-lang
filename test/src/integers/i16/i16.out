4
5
6000
